core/bones/spatial.py

```diff
@@ -54,10 +54,16 @@ def __init__(self, *, boundsLat: Tuple[int, int], boundsLng: Tuple[int, int], gr
             :param gridDimensions: Number of sub-regions the map will be divided in
         """
         super().__init__(**kwargs)
-        assert isinstance(boundsLat, tuple) and len(boundsLat) == 2, "boundsLat must be a tuple of (int, int)"
-        assert isinstance(boundsLng, tuple) and len(boundsLng) == 2, "boundsLng must be a tuple of (int, int)"
+        assert isinstance(boundsLat, tuple) and len(boundsLat) == 2, "boundsLat must be a tuple of (float, float)"
+        assert isinstance(boundsLng, tuple) and len(boundsLng) == 2, "boundsLng must be a tuple of (float, float)"
         assert isinstance(gridDimensions, tuple) and len(
             gridDimensions) == 2, "gridDimensions must be a tuple of (int, int)"
+        # Checks if boundsLat and boundsLng have possible values
+        # See https://docs.mapbox.com/help/glossary/lat-lon/
+        assert boundsLat[0] >= -90 and boundsLat[0] <= 90, "boundsLat[0] must be between -90 and 90"
+        assert boundsLat[1] >= -90 and boundsLat[1] <= 90, "boundsLat[1] must be between -90 and 90"
+        assert boundsLng[0] >= -180 and boundsLng[0] <= 180, "boundsLng[0] must be between -180 and 180"
+        assert boundsLng[1] >= -180 and boundsLng[1] <= 180, "boundsLng[1] must be between -180 and 180"
```

Comment:
I think, as in some other places in ViUR, the use of `assert` is wrong here. `assert` is a statement for debugging purposes and making internal assumptions, not for validating method arguments.

```suggestion
        if not -90 <= boundsLat[0] <= 90:
			raise ValueError(f"boundsLat[0] must be between -90 and 90. Got {boundsLat[0]!r}")
        if not -90 <= boundsLat[1] <= 90:
			raise ValueError(f"boundsLat[1] must be between -90 and 90. Got {boundsLat[1]!r}")
        if not -180 <= boundsLng[0] <= 180:
			raise ValueError(f"boundsLng[0] must be between -180 and 180. Got {boundsLng[0]!r}")
        if not -180 <= boundsLng[1] <= 180:
			raise ValueError(f"boundsLng[1] must be between -180 and 180. Got {boundsLng[1]!r}")
```

For reference:
https://docs.python.org/3/reference/simple_stmts.html#assert
https://dbader.org/blog/python-assert-tutorial